pjsip/src/pjsua-lib/pjsua_media.c

```diff
@@ -3165,8 +3165,7 @@ static void stop_media_stream(pjsua_call *call, unsigned med_idx)
 static void stop_media_session(pjsua_call_id call_id)
 {
     pjsua_call *call = &pjsua_var.calls[call_id];
-    unsigned mi;
-
+    unsigned mi;   
```

Comment:
We put local variables in the beginning of function (FYI, we tried our best to follow ANSI C for portability, [see here](https://docs.pjsip.org/en/latest/get-started/coding-style.html#use-ansi-c-c90-standard)), not before its usage. So the original is actually better.